addons/account/models/res_partner_bank.py

```diff
@@ -16,14 +17,33 @@ class ResPartnerBank(models.Model):
     journal_id = fields.One2many(
         'account.journal', 'bank_account_id', domain=[('type', '=', 'bank')], string='Account Journal', readonly=True,
         help="The accounting journal corresponding to this bank account.")
+    iban_warning = fields.Html(
+        compute='_compute_display_account_warning',
+        help='Technical field used to display a warning if the IBAN country is different than the holder country.',
+        store=True,
+    )
+    money_transfer_warning = fields.Html(
+        compute='_compute_display_account_warning',
+        help='Technical field used to display a warning if the account is a transfer service account.',
+        store=True,
+    )
+    partner_supplier_rank = fields.Integer(related='partner_id.supplier_rank')
+    partner_customer_rank = fields.Integer(related='partner_id.customer_rank')
+    related_moves = fields.One2many('account.move', inverse_name='partner_bank_id')
 
     # Add tracking to the base fields
     bank_id = fields.Many2one(tracking=True)
     active = fields.Boolean(tracking=True)
     acc_number = fields.Char(tracking=True)
     acc_holder_name = fields.Char(tracking=True)
     partner_id = fields.Many2one(tracking=True)
-    allow_out_payment = fields.Boolean(tracking=True)
+    user_has_group_validate_bank_account = fields.Boolean(compute='_compute_user_has_group_validate_bank_account')
+    allow_out_payment = fields.Boolean(
+        tracking=True,
+        help='Sending fake invoices with fraudulent account number is a common phishing practice.'
+             'To protect yourself, always verify new bank account numbers, preferably by calling the vendor as phishing '
+             'usually happens when their emails are compromised. Once verified, you can activate the ability to send money.'
```

Comment:
- missing space at the end of the first sentence
- it is "invoices with **a** fraudulent account number..." (or possibly "invoices with fraudulent account number**s**...")
- missing comma after "vendor"

```suggestion
        help='Sending fake invoices with a fraudulent account number is a common phishing practice. '
             'To protect yourself, always verify new bank account numbers, preferably by calling the vendor, as phishing '
             'usually happens when their emails are compromised. Once verified, you can activate the ability to send money.'
```